src/components/reports/ReportForm.tsx

```diff
@@ -63,10 +63,48 @@ const ReportForm: FC<Props> = (props: Props) => {
     const bundle = bundleRequest(values, reportedGenes);
 
     setResult(JSON.stringify(JSON.parse(bundle.body), null, 2));
+    const resourceList = JSON.parse(bundle.body).entry.map((entry: any) => entry.resource.resourceType);
 
     ctx.client
       ?.request(bundle)
-      .then((response) => console.debug("Bundle submitted", bundle, response))
+      .then((response) => {
+        console.debug("Bundle submitted", bundle, response);
+        const errors = getErrors(response, resourceList);
+        if (errors.length > 1) {
+          const errorsTable = () => {
+            //originally had this in ReportForm.module.css but didn't work?
+            const css = ` 
+            .errors-table {
+              font: inherit;
+              font-size: 86%;
+            }`;
+            return (
+              <>
+                <style>{css}</style>
+                <table className="errors-table">
+                  <thead className="errors-table__header">
+                    <tr>
+                      <th>Code</th>
+                      <th>Resource</th>
+                      <th>Information</th>
+                    </tr>
+                  </thead>
+                  <tbody className="errors-table__body">
+                    {errors.map((error, i) => (
+                      <tr key={i}>
+                        <td>{error.errorCode}</td>
+                        <td>{error.resourceType}</td>
+                        <td>{error.diagnostics}</td>
+                      </tr>
+                    ))}
+                  </tbody>
+                </table>
+              </>
+            );
+          };
```

Comment:
I'm not sure what this is referring to specifically since this is pointing at the mapping and I'm not sure how to loop through all the items in the array and display them in a table without the method, what am I missing?

Reply:
I would have thought that this should just work, but I haven't tried it so maybe I'm missing something?
```suggestion
          const errorsTable = (
              <>
                 <table className={classes["errors-table"]}>
                    <tr>
                      <th>Code</th>
                      <th>Resource</th>
                      <th>Information</th>
                    </tr>
                  </thead>
                  <tbody className="errors-table__body">
                    {errors.map((error, i) => (
                      <tr key={i}>
                        <td>{error.errorCode}</td>
                        <td>{error.resourceType}</td>
                        <td>{error.diagnostics}</td>
                      </tr>
                    ))}
                  </tbody>
                </table>
              </>
            );
          };
```

Reply:
aaah okay, you meant the higher level (overall table). I'll try it and see if the tests are still happy

Reply:
the test for it doesn't seem to find it but seeing as I've been having trouble triggering the error modal in the browser I can't say definitively that it doesn't work

Reply:
If you're having problems with it, I'd suggest reverting this section to where it was working and then see if that fixes it. If you're still having the same problem then that suggests that it's not that change which was affected it. If it does appear then try adding in the changes and see if that breaks it again

Reply:
it works with the old method of finding the error modal (before the `await waitFor`) so I'm happy to keep the change